docs/make.jl

```diff
@@ -60,11 +66,10 @@ pages["Model Library"] = make_model_library(
 # section name should be the name of the file for instance network_matrices.jl -> Network Matrices
 julia_file_filter = x -> occursin(".jl", x)
 folders = Dict(
-    "User Guide" => filter(julia_file_filter, readdir("docs/src/user_guide")),
     "Model Library" => filter(julia_file_filter, readdir("docs/src/model_library")),
```

Comment:
I think that the Model Library and its sections could use some introductory text.

Reply:
Can you provide more specifics?